lib/pure/times.nim

```diff
@@ -92,80 +90,52 @@ elif defined(windows):
   type
     Time* = distinct TimeImpl
 
+type
+  Month* = enum ## Represents a month. Note that the enum starts at ``1``, so ``ord(month)`` will give
+                ## the month number in the range ``[1..12]``.
+    mJan = 1, mFeb, mMar, mApr, mMay, mJun, mJul, mAug, mSep, mOct, mNov, mDec
```

Comment:
This is problematic for today's Nim. This means ``Month`` is then marked as  a type that needs explicit initialization.

Reply:
It's a bit annoying, but I think `Month` is rarely used as a standalone type anyway. Having it start at 1 is a lot more natural, since that's generally what's wanted when converting to an `int`.